pipenv/core.py

```diff
@@ -288,7 +288,6 @@ def ensure_pipfile(validate=True, skip_requirements=False):
     if validate and project.virtualenv_exists and not PIPENV_SKIP_VALIDATION:
         # Ensure that Pipfile is using proper casing.
         p = project.parsed_pipfile
-        p.clear_pipfile_cache()
```

Comment:
This line was buggy but was never hit because `PIPENV_SKIP_VALIDATION` is always true